### PR TITLE
Simplify two if (remove redundant code)

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Remove redundant code

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/etcd/local.go [hunk 1]
```diff
@@ -95,18 +95,15 @@ func RemoveStackedEtcdMemberFromCluster(client clientset.Interface, cfg *kubeadm
...
-	if err != nil {
-		return err
-	}
-
-	klog.V(1).Infof("[etcd] removing etcd member: %s, id: %d", etcdPeerAddress, id)
-	members, err := etcdClient.RemoveMember(id)
-	if err != nil {
+	if id, err := etcdClient.GetMemberID(etcdPeerAddress); err != nil {
 		return err
+	} else {
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/etcd/local.go]: this change will probably not pass golint [1]
please take a look at the CI report.

File: cmd/kubeadm/app/phases/etcd/local.go [hunk 2]
```diff
@@ -116,32 +113,29 @@ func RemoveStackedEtcdMemberFromCluster(client clientset.Interface, cfg *kubeadm
...
-	etcdPeerAddress := etcdutil.GetPeerURL(endpoint)
-
-	klog.V(1).Infof("Adding etcd member: %s", etcdPeerAddress)
-	initialCluster, err := etcdClient.AddMember(nodeName, etcdPeerAddress)
-	if err != nil {
-		return err
-	}
-	fmt.Println("[etcd] Announced new etcd member joining to the existing etcd cluster")
-	klog.V(1).Infof("Updated etcd member list: %v", initialCluster)
+	} else {
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/etcd/local.go]: [1]

File: cmd/kubeadm/app/phases/etcd/local.go [hunk 3]
```diff
@@ -95,18 +95,16 @@ func RemoveStackedEtcdMemberFromCluster(client clientset.Interface, cfg *kubeadm
...
 	klog.V(2).Infof("[etcd] get the member id from peer: %s", etcdPeerAddress)
-	id, err := etcdClient.GetMemberID(etcdPeerAddress)
-	if err != nil {
-		return err
-	}
-
-	klog.V(1).Infof("[etcd] removing etcd member: %s, id: %d", etcdPeerAddress, id)
-	members, err := etcdClient.RemoveMember(id)
-	if err != nil {
+	if id, err := etcdClient.GetMemberID(etcdPeerAddress); err != nil {
```
Comment (SataQiu) [inline cmd/kubeadm/app/phases/etcd/local.go]: Do we need to do this? 
/cc @neolit123

**Review by neolit123 — COMMENTED** [review]
thanks for the PR @xichengliudui 
but i don't think the linters will be happy with this change and also it seems more like code shuffle instead of removing redundant code. :)

**Pull request closed without merging**